config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+	get '/guests/:id', to: 'guests#show'
+
+	post '/room_guests/:guest_id', to: 'room_guests#create'
```

Comment:
I agree with the verb, controller, and action for this route, but usually when we are `POST`-ing to a resource, the ID of that resource will not have been created yet, so it won't have an ID in the URI. 

If you think about the request-response cycle and the rules of RESTful routes, the verb should match with the path and the action in the controller it's being routed to. If we're creating a new resource called `RoomGuest`, then the necessary attributes will be part of that POST data, in the request body. 
Typically, a URI will have an ID that corresponds with its previous string; so an ID on this route would generally be used for a `RoomGuest.id` attribute. Since you're sending in a `guest_id` here, it can easily get confused with a different kind of ID if you just look at the path itself.

So, my suggestion is to put this `guest_id` inside the form body, and keep everything else the same. You'll still have access to it in your params. Alternatively, your route could look like: 
```
post "/guests/:guest_id/rooms", to: "room_guests#create"
```
That way we know we're still creating a relatonship to this room from this guest, and doing it in the place that makes sense, but we are specific about what kind of ID this is.